src/types.ts

```diff
@@ -1 +1,4 @@
-export type ThemeColorOption = "white" | "gray";
+type ThemeColorOption = "white" | "gray" | "blue";
+type TitleColorOption = "blue";
+
+export { ThemeColorOption, TitleColorOption };
```

Comment:
Nit: I'd personally prefer to leave the `export`s on the lines that initially define the values, especially in cases like this where we're exporting every named thing in a file. In certain cases, it may be useful to keep all the exports in one place for describing what the "public" API of a file is in one place, but for files that are relatively short or that are relatively transparent, it causes more indirection for a reader to separate `export`s from the initial declarations.

I acknowledge that this is really just a matter of style and that we don't have any explicit guidance about this written down anywhere, so I'm not adamant about moving the `export`s to the declarations.

cc @jessica-px, in case she has any opinions about this.